sdk/iot/hub/samples/src/paho_iot_hub_twin_example.c

```diff
@@ -310,13 +310,11 @@ static int send_get_twin()
 static int build_reported_property(az_json_builder* json_builder)
 {
   az_result result;
-  result = az_json_builder_init(json_builder, AZ_SPAN_FROM_BUFFER(reported_property_payload));
-  result = az_json_builder_append_token(json_builder, az_json_token_object_start());
-  az_json_token reported_property_value_token
-      = az_json_token_number((double)reported_property_value++);
-  result = az_json_builder_append_object(
-      json_builder, reported_property_name, reported_property_value_token);
-  result = az_json_builder_append_token(json_builder, az_json_token_object_end());
+  result = az_json_builder_init(json_builder, AZ_SPAN_FROM_BUFFER(reported_property_payload), NULL);
+  result = az_json_builder_append_begin_object(json_builder);
+  result = az_json_builder_append_property_name(json_builder, reported_property_name);
+  result = az_json_builder_append_int32_number(json_builder, reported_property_value++);
```

Comment:
Fyi, @CIPop, @danewalton - updating the sample based on recent changes to JSON builder.